source/WebApp/components/internal/codemirror/mode-asm-instructions.ts

```diff
@@ -0,0 +1,2 @@
?:1(?:23p[ds]|32p[ds])|2(?:13p[ds]|31p[ds])|3(?:12p[ds]|21p[ds])|p[ds])))|ixupimm(?:p[ds]|s[ds])|nm(?:add(?:1(?:23(?:p[ds]|s[ds])|32(?:p[ds]|s[ds]))|2(?:13(?:p[ds]|s[ds])|31(?:p[ds]|s[ds]))|3(?:12(?:p[ds]|s[ds])|21(?:p[ds]|s[ds]))|p[ds]|s[ds])|sub(?:1(?:23(?:p[ds]|s[ds])|32(?:p[ds]|s[ds]))|2(?:13(?:p[ds]|s[ds])|31(?:p[ds]|s[ds]))|3(?:12(?:p[ds]|s[ds])|21(?:p[ds]|s[ds]))|p[ds]|s[ds]))|pclass(?:p[ds]|s[ds])|rcz(?:p[ds]|s[ds]))|g(?:ather(?:dp[ds]|pf(?:0(?:dp[ds]|qp[ds])|1(?:dp[ds]|qp[ds]))|qp[ds])|et(?:exp(?:p[ds]|s[ds])|mant(?:p[ds]|s[ds])))|h(?:addp[ds]|subp[ds])|m(?:a(?:x(?:p[ds]|s[ds])|skmov(?:p[ds]|dqu))|c(?:all|lear)|l(?:aunch|oad)|p(?:sadbw|tr(?:ld|st))|r(?:e(?:ad|sume)|un)|func|in(?:p[ds]|s[ds])|mcall|ov(?:d(?:|q(?:a(?:|32|64)|u(?:|16|32|64|8))|dup)|h(?:p[ds]|lps)|l(?:p[ds]|hps)|q(?:|q[au])|s(?:d|hdup|ldup|s)|ap[ds]|mskp[ds]|nt(?:p[ds]|dq(?:|a)|qq)|up[ds])|save|ul(?:p[ds]|s[ds])|write|xo(?:ff|n))|p(?:a(?:bs[bdqw]|ck(?:ss(?:dw|wb)|us(?:dw|wb))|dd(?:s[bw]|b|d|q|us[bw]|w)|lignr|nd(?:|n(?:|d|q)|d|q)|vg[bw])|b(?:lend(?:m[bdqw]|d|vb|w)|roadcast(?:m(?:b2q|w2d)|b|d|q|w))|c(?:m(?:p(?:e(?:q[bdqw]|str[im])|u[bdqw]|b|d|gt[bdqw]|istr[im]|q|w)|ov)|o(?:m(?:u[bdqw]|b|d|press[dq]|q|w)|nflict[dq])|lmul(?:hq(?:hqdq|lqdq)|lq(?:hqdq|lqdq)|qdq))|e(?:x(?:pand[dq]|tr[bdqw])|rm(?:2(?:f128|i128)|i(?:2(?:p[ds]|b|d|q|w)|lp[ds])|p[ds]|b|d|q|t2(?:p[ds]|b|d|q|w)|w))|h(?:add(?:b[dqw]|d(?:|q)|u(?:b[dqw]|w[dq]|dq)|w(?:|d|q)|sw)|minposuw|sub(?:d(?:|q)|w(?:|d)|bw|sw))|m(?:a(?:d(?:d(?:52(?:huq|luq)|ubsw|wd)|cs(?:swd|wd))|x(?:s[bdqw]|u[bdqw])|cs(?:d(?:q[hl]|d)|s(?:d(?:q[hl]|d)|w[dw])|w[dw])|skmov[dq])|in(?:s[bdqw]|u[bdqw])|ov(?:d(?:2m|b|w)|m(?:2[bdqw]|skb)|q(?:2m|b|d|w)|s(?:d[bw]|q[bdw]|x(?:b[dqw]|w[dq]|dq)|wb)|w(?:2m|b)|b2m|us(?:d[bw]|q[bdw]|wb)|zx(?:b[dqw]|w[dq]|dq))|ul(?:h(?:rsw|uw|w)|l[dqw]|dq|tishiftqb|udq))|s(?:h(?:a[bdqw]|l[bdqw]|uf(?:b|d|hw|lw))|r(?:a(?:v[dqw]|d|q|w)|l(?:d(?:|q)|v[dqw]|q|w))|adbw|catter(?:d[dq]|q[dq])|ign[bdw]|ll(?:d(?:|q)|v[dqw]|q|w)|ub(?:s[bw]|b|d|q|us[bw]|w))|gather(?:d[dq]|q[dq])|insr[bdqw]|lzcnt[dq]|or(?:|d|q)|perm|ro(?:l(?:v[dq]|d|q)|r(?:v[dq]|d|q)|t[bdqw])|te(?:rnlog[dq]|st(?:|m[bdqw]|nm[bdqw]))|unpck(?:h(?:bw|dq|qdq|wd)|l(?:bw|dq|qdq|wd))|xor(?:|d|q))|r(?:ange(?:p[ds]|s[ds])|cp(?:14(?:p[ds]|s[ds])|28(?:p[ds]|s[ds])|ps|ss)|educe(?:p[ds]|s[ds])|ndscale(?:p[ds]|s[ds])|ound(?:p[ds]|s[ds])|sqrt(?:14(?:p[ds]|s[ds])|28(?:p[ds]|s[ds])|ps|ss))|s(?:ca(?:lef(?:p[ds]|s[ds])|tter(?:dp[ds]|pf(?:0(?:dp[ds]|qp[ds])|1(?:dp[ds]|qp[ds]))|qp[ds]))|huf(?:f(?:32x4|64x2)|i(?:32x4|64x2)|p[ds])|qrt(?:p[ds]|s[ds])|tmxcsr|ub(?:p[ds]|s[ds]))|u(?:comis[ds]|npck(?:hp[ds]|lp[ds]))|insert(?:f(?:128|32x[48]|64x[24])|i(?:128|32x[48]|64x[24])|ps)|ld(?:dqu|mxcsr|qqu)|orp[ds]|testp[ds]|xorp[ds]|zero(?:all|upper))|w(?:r(?:fsbase|gsbase|msr|pkru|shr)|ait|binvd)|x(?:a(?:bort|cquire|dd)|b(?:egin|ts)|c(?:hg|rypt(?:c(?:bc|fb|tr)|ecb|ofb))|r(?:elease|stor(?:|s(?:|64)|64))|s(?:ave(?:|c(?:|64)|s(?:|64)|64|opt(?:|64))|etbv|ha(?:1|256)|tore)|end|getbv|lat(?:|b)|or(?:|p[ds])|test)|zeroupper)/);
```

Comment:
1. You can use `export default new RegExp(...);` here, then you can just do `import instructionsRegex from ...` when importing.
2. Since you already have `/.../`, this already generates a RegExp instance, so I don't think you need `new RegExp()` around it.

Reply:
True, RegExp ctor is useful when we have to use variables. In this case, we need literals as it is a true constant at build time. :)